### PR TITLE
Add system test for 'Sink.update'.

### PR DESCRIPTION
**Pull request by tseaver**

### CODE REVIEW

File: system_tests/logging_.py [hunk 1]
```diff
@@ -193,3 +197,18 @@ def test_reload_sink(self):
         sink.reload()
         self.assertEqual(sink.filter_, DEFAULT_FILTER)
         self.assertEqual(sink.destination, uri)
+
+    def test_update_sink(self):
+        bucket_uri = self._init_storage_bucket()
+        dataset_uri = self._init_bigquery_dataset()
+        UPDATED_FILTER = 'logName:syslog'
```
Comment (dhermes) [inline system_tests/logging_.py]: What does this log name correspond to?

Comment (tseaver) [inline system_tests/logging_.py]: It is an arbitrary filter expression, limiting entries exported via the sink to those whose `logName` field matches that string.  We use a narrower filter (`logName:syslog AND severity:ERROR`) as the default filter.

**Comment by dhermes** [comment]
LGTM

**Comment by tseaver** [comment]
Travis is dawwwwg-slow today.

**Comment by tseaver** [comment]
I pushed this change 'Two hours ago' -- still no Travis love?

**Pull request merged**